charts/timescaledb-single/templates/configmap-scripts.yaml

```diff
@@ -69,6 +69,28 @@ data:
     [ -d "${WALDIR}" ] || install -o postgres -g postgres -d -m 0700 "${WALDIR}"
 
     pgbackrest --stanza=poddb --force --delta --log-level-console=detail restore
+  restore_or_initdb.sh: |
```

Comment:
We'll be adding restore capabilities to this script, so that's the reason for the name